src/app/alerts/alert.service.ts

```diff
@@ -33,7 +33,9 @@ export class AlertService {
 
   addAlert(alert: Alert) {
     this.alerts.push(alert);
-    this.openSnackBar(alert);
+    if (!alert.silent) {
+      this.openSnackBar(alert);
```

Comment:
I am submitting a commit, refactoring the silent and the switch-case to decide the display of alerts into one system that's more explicit than the original `openSnackBar()` function